### ISSUE TITLE
Keymaps regression

### ISSUE THREAD

**Issue by bet4it** [zeit/hyper#4173]
This bug was introduced in #3957.

We need to init `Mousetrap` with `true` as the second argument: 
https://github.com/zeit/hyper/blob/d5d5bcc74ca8de4c568c410fe042b5ef353baf67/lib/containers/hyper.js#L58
But we remove this argument after #3957:
https://github.com/zeit/hyper/blob/a1eb84d8a7c277ddf29d739e46cde14c4c14b4f7/lib/containers/hyper.tsx#L62

Refer: ccampbell/mousetrap#400

**Comment by bet4it** [comment]
@LabhanshAgrawal

**Comment by LabhanshAgrawal** [comment]
Oh, iirc this was changed after adding the mousetrap type definitions

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f4f29cb4f9f25905a01bfa319a434378282dc6c5/types/mousetrap/index.d.ts#L13

Since it only listed 1 argument of Element type (optional which defaults to document) I removed those and tested, but didn't face any issues. I see, we are using a fork of Mousetrap which adds the capture option.

We can change this, and add a type declaration override in the ext-modules file.

P.S. what issue is caused by this? I didn't face any.
